pkg/interface/webterm/App.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {
+  useCallback, useEffect
+} from 'react';
+
+import useTermState from './state';
+import { useDark } from './lib/useDark';
+import api from './api';
+
+import { _dark, _light } from '@tlon/indigo-react';
+
+import 'xterm/css/xterm.css';
+
+import {
+  scrySessions
+} from '@urbit/api/term';
+
+import { ThemeProvider } from 'styled-components';
+import { Tabs } from './Tabs';
+import Buffer from './Buffer';
+import { DEFAULT_SESSION } from './constants';
+import { showSlog } from './lib/blit';
+import { InfoButton } from './InfoButton';
+
+export default function TermApp() {
+  const { names, selected } = useTermState();
+  const dark = useDark();
+
+  const initSessions = useCallback(async () => {
```

Comment:
this doesn't need to be a closure i.e. you can pull it's definition outside the component